### ISSUE TITLE
Wrong API Endpoint

### ISSUE THREAD

**Issue by brianjhanson** [nystudio107/craft-imageoptimize-imgix#4]
### Describe the bug
This morning we were debugging an issue that was causing Image Optimize to warn us that our API key was deprecated despite using the new Imgix format (pretty much the same issue as [#273](https://github.com/nystudio107/craft-imageoptimize/issues/273)). After a bit of digging, I found this in the logs

```log
2021-12-02 08:24:17 [-][1][-][error][nystudio107\imageoptimizeimgix\imagetransforms\ImgixImageTransform::purgeUrl] Error purging URL: https://subdomain.imgix.net/folder/filename.jpg - Client error: `POST https://api.imgix.com/api/v1/purge` resulted in a `401 Unauthorized` response:
{
  "errors": [
    {
      "detail": "A valid API key is required and could not be determined.",
      "id": "0d0a28ccf (truncated...)
```

I was pretty sure the API key was valid, so I did a quick test directly against the Imgix API and noticed that the [URL the package seems to be using](https://github.com/nystudio107/craft-imageoptimize-imgix/blob/6c1fa8c80e4fc4d457f93abb3cc23bb171126898/src/imagetransforms/ImgixImageTransform.php#L46) (assuming I'm reading things right) doesn't match the URL in the error message or URL in the [imgix docs](https://docs.imgix.com/apis/management#purge-examples). 

Sure enough, when I make a request directly to Imgix with the `IMGIX_PURGE_ENDPOINT` I get a 404, but when I use `IMGIX_PURGE_ENDPOINT_OLD` things work as expected.

Here's what worked for me:
```
POST https://api.imgix.com/api/v1/purge
Authorization: Bearer {token}
Content-Type: application/json

{
  "data": {
    "attributes": {
      "url": "https://source.imgix.net/folder/filename.jpg"
    },
    "type": "purges"
  }
}
```

### To reproduce
1. Upload an image to Craft
2. Use the "replace image" functionality to replace the image
3. Error should be in the logs

### Expected behavior
Image should be purged from Imgix and key should register as used.

### Versions
**Plugin Versions**
<table class="data fullwidth fixed-layout" dir="ltr">
            <tbody>
                                    <tr>
                        <th class="light">Amazon S3</th>
                        <td>1.3.0</td>
                    </tr>
                                    <tr>
                        <th class="light">Asset Rev</th>
                        <td>6.0.2</td>
                    </tr>
                                    <tr>
                        <th class="light">Blitz</th>
                        <td>3.11.0</td>
                    </tr>
                                    <tr>
                        <th class="light">greenhouse-jobs</th>
                        <td>1.3.0</td>
                    </tr>
                                    <tr>
                        <th class="light">ImageOptimize</th>
                        <td>1.6.38</td>
                    </tr>
                                    <tr>
                        <th class="light">Neo</th>
                        <td>2.11.19</td>
                    </tr>
                                    <tr>
                        <th class="light">Redactor</th>
                        <td>2.8.8</td>
                    </tr>
                                    <tr>
                        <th class="light">SEOmatic</th>
                        <td>3.4.20</td>
                    </tr>
                                    <tr>
                        <th class="light">Typed link field</th>
                        <td>1.0.25</td>
                    </tr>
                                    <tr>
                        <th class="light">Typogrify</th>
                        <td>1.1.18</td>
                    </tr>
                            </tbody>
        </table>

**Craft Version**
3.7.23

**Comment by brianjhanson** [comment]
Any update on this? Or can you point me towards what might be happening? It feels like this would be a pretty wide reaching bug which is why I think it's probably something we're doing. 

Although on the other hand, we use ImageOptimize on a handful of sites and I have noticed that none of them seem to  actually be using their Imgix API keys

**Comment by khalwat** [comment]
Very strange, I will have a look at this shortly. ImageOptimize only uses the API key to do purging, so if that's not working, it'd make sense.

**Comment by khalwat** [comment]
Okay, so the issue is a little bit confusing in terms of Imgix... they deprecated their `v2` API and replaced it with the `v1` API, at a different endpoint... which lead to me confusing the two.

Addressed in: https://github.com/nystudio107/craft-imageoptimize-imgix/commit/6f64dc5acda484625e3c4d727fe8f595e532f259

**Comment by brianjhanson** [comment]
@khalwat awesome, thank you! For what it's worth, I was equally confused when I was trying to track this down

**Comment by khalwat** [comment]
Released as 1.6.42: https://github.com/nystudio107/craft-imageoptimize/releases/tag/1.6.42
